flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/plan/rules/physical/stream/StreamPhysicalPythonGroupWindowAggregateRule.java

```diff
@@ -77,7 +77,11 @@ public boolean matches(RelOptRuleCall call) {
                 aggCalls.stream()
                         .anyMatch(x -> PythonUtil.isPythonAggregate(x, PythonFunctionKind.PANDAS));
         boolean existJavaFunction =
-                aggCalls.stream().anyMatch(x -> !PythonUtil.isPythonAggregate(x, null));
+                aggCalls.stream()
```

Comment:
Does the issue also exist in StreamPhysicalPythonOverAggregateRule, BatchPhysicalPythonAggregateRule, BatchPhysicalPythonWindowAggregateRule?

Reply:
Currently we doesn't support general python udaf in stream over window and batch mode.